lib/teleterm/daemon/daemon.go

```diff
@@ -822,8 +823,10 @@ func (s *Service) ListKubernetesResources(ctx context.Context, clusterURI uri.Re
 			UseSearchAsRoles:    req.GetUseSearchAsRoles(),
 			KubernetesCluster:   req.GetKubernetesCluster(),
 			KubernetesNamespace: req.GetKubernetesNamespace(),
-			TeleportCluster:     cluster.Name,
-		})
+			TeleportCluster:     tc.SiteName,
+		}
+
+		resources, err = apiclient.GetKubernetesResourcesWithFilters(ctx, kubeServiceClient, req)
```

Comment:
Just curious, what's the reason behind this change?

Reply:
`kubernetesServiceClient.ListKubernetesResources` fetches just a single page of resources. `api/client.GetKubernetesResourcesWithFilters` fetches everything.